src/cryptonote_core/blockchain.cpp

```diff
@@ -2618,14 +2618,14 @@ bool Blockchain::get_split_transactions_blobs(const std::vector<crypto::hash>& t
       cryptonote::blobdata tx;
       if (m_db->get_pruned_tx_blob(tx_hash, tx))
       {
-        txs.emplace_back(tx_hash, std::move(tx), crypto::null_hash, cryptonote::blobdata());
-        if (!is_v1_tx(std::get<1>(txs.back())) && !m_db->get_prunable_tx_hash(tx_hash, std::get<2>(txs.back())))
+        auto& [hash, pruned, pruned_hash, prunable] = txs.emplace_back(tx_hash, std::move(tx), crypto::null_hash, cryptonote::blobdata());
```

Comment:
much easier to read :+1:

Reply:
Multiple C++17isms happening here at once.